src/backend/Dockerfile.py-api

```diff
@@ -6,6 +6,15 @@
 FROM public-registry.caliopen.org/caliopen_python
 MAINTAINER Caliopen
 
+RUN apk add nginx linux-headers supervisor
+RUN pip install --upgrade pip
+RUN pip install uwsgi
+
+# Copy configuration files to run apiv1 behind uwsi and nginx
+COPY ./configs/apiv1-nginx.conf /etc/nginx/nginx.conf
+COPY ./configs/apiv1-uwsgi.ini /etc/uwsgi/uwsgi.ini
+COPY ./configs/apiv1-supervisord.conf /etc/supervisor.d/apiv1.ini
```

Comment:
Can't we mount these like we do with the rest of the configuration files?

Reply:
In a previous version it's what I did, but I think it's better to build the image this way. We can mount another version of one of this file in the cluster that will overwrite this version for 'development'.

Doing this way the image is more independant than the infrastructure described in the compose file.